Cargo.lock

```diff
@@ -1802,20 +1810,74 @@ version = "0.9.0+wasi-snapshot-preview1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "cccddf32554fecc6acb585f82a32a72e28b48f8c4c1883ddfeeeaa96f7d8e519"
 
+[[package]]
+name = "wasm-bindgen"
+version = "0.2.59"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3557c397ab5a8e347d434782bcd31fc1483d927a6826804cec05cc792ee2519d"
+dependencies = [
+ "cfg-if",
+ "wasm-bindgen-macro",
+]
```

Comment:
Again wasm-bindgen here??

Reply:
This is from criterion via plotters via js-sys, I believe it should be a dev-dependency

Reply:
Which, from their Cargo.toml, seems to be only for `[target.'cfg(target_arch = "wasm32")'.dependencies]`